bigquery_etl/query_scheduling/templates/airflow_dag.j2

```diff
@@ -85,9 +85,10 @@ with DAG('{{ name }}', default_args=default_args{%+ if schedule_interval != None
             {%+ if task.email_on_retry != None -%}
             email_on_retry={{ task.email_on_retry }},
             {%+ endif -%}
+    {% if not task.is_python_script -%}
             dag=dag,
-        )
     {% endif -%}
+        )
```

Comment:
this change ensures retry properties are included for both python scripts and queries.